benchmarks/verifications.py

```
@@ -0,0 +1,474 @@
+def verify_helloworld_object(json_object, url):
+    '''
+    Ensure that the JSON object closely resembles
+    { 'message': 'Hello, World!' }
+    '''
+
+    problems = []
+
+    try:
+        # Make everything case insensitive
+        json_object = {k.lower(): v.lower() for k, v in json_object.items()}
+    except:
```

Ruff

Do not use bare `except`

(E722)